### ISSUE TITLE
Property sets don't stick when attached to an element

### ISSUE THREAD

**Issue by BobRay** [modxbot/migrate#1422]
bobray created Redmine issue ID 1422

If you attach a property set to an element on the Property Sets panel, it doesn't show up on the Properties tab for the element. If you attach a property set on the Properties tab, it does show up on the Property Sets Panel as attached, but when you navigate away and return to the element, the property set is gone from the Properties Tab (it remains on the Property Sets Panel). Also, when you have multiple property sets (some with elements attached and some without) and try to attach one on the Properties tab of an element, not all of them show in the drop-down list. I can't see any pattern to which ones show and which don't.

**Comment by BobRay** [comment]
bobray submitted:

Installing SPForm creates a property set that shows on SPForm's Properties tab but doesn't show up on the Property Sets screen.

**Comment by splittingred** [comment]
splittingred submitted:

Bob, I can't replicate anything you've just described. 1. I attached an Element to a Property Set via the Property Sets page, then viewed the Element - and the Property Set was available in the dropdown.
2. I attached a Property Set to an Element via that Element's Properties tab, and then went to the Property Set page, and the Element showed as attached there.
3. I attached multiple Sets to an Element, and all showed up fine in the dropdown.
